executor/tests/manual/test_upgrade_local.py

```
@@ -111,8 +112,7 @@ def main():
     """主测试函数"""
     parser = argparse.ArgumentParser(description="测试执行器自更新功能")
     parser.add_argument(
-        "--full", action="store_true",
-        help="运行完整更新流程测试（会调用API）"
+        "--full", action="store_true", help="运行完整更新流程测试（会调用API）"
```

Ruff (0.15.6)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)